README.md

```diff
@@ -67,6 +67,12 @@ Supported options:
 - `phone` - Previously verified phone number for a user.
 - `customerUid` - An ID or identifier for the user in your system.
 - `templateKey` - The template key for this transaction.
+- `matchData` - The array of data to compare against the extracted data
+- - `givenName` - given name to match against
+- - `familyName` - family name to match against
+- - `middleName` - middle name to match against
+- - `dateOfBirth` - date of birth to match against
+- - `idExpiryDate` - id expiry date to match against
```

Comment:
is this available via the API, or only via the PHP SDK. Just wondering if we need to update the docs on readme.io now aw well